<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TileViewer</class>
 <widget class="QDialog" name="TileViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>756</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>4</number>
     </property>
     <widget class="QWidget" name="layoutWidget1">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="60,1">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="minimumSize">
              <size>
               <width>512</width>
               <height>256</height>
              </size>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>774</width>
                <height>356</height>
               </rect>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Zoom</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="zoomLevel">
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="minimum">
                <double>1.000000000000000</double>
               </property>
               <property name="maximum">
                <double>16.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.250000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_drawgrid">
               <property name="text">
                <string>Draw grid</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_mouseover">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QFrame" name="frame">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>50</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="lineWidth">
                <number>2</number>
               </property>
               <property name="midLineWidth">
                <number>1</number>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QLabel" name="label_characterimage">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>64</width>
                    <height>64</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Pattern:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="label_charpat">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Address</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="label_charadr">
                 <property name="text">
                  <string>PGT: $0348, CT: $200D
NT: $196E (base+$16E)</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_highlight">
               <property name="toolTip">
                <string>visible when using  'Name Table + Overlay' as Tilemap Source</string>
               </property>
               <property name="text">
                <string>Highlight Tile</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="sp_highlight">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_highligtinfo">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>($DD, not in NT)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="refreshButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Take VRAM snapshot</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Tilemap Source:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="cb_tilemapsource">
                   <item>
                    <property name="text">
                     <string>Pattern Table</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Name Table</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Name Table + Overlay</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_M5">
                 <property name="text">
                  <string>M5</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_M4">
                 <property name="text">
                  <string>M4</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_M3">
                 <property name="text">
                  <string>M3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_M2">
                 <property name="text">
                  <string>M2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_M1">
                 <property name="text">
                  <string>M1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>=&gt; Mode </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_screenmode">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="useVDPRegisters">
               <property name="text">
                <string>Use current VDP settings</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Screen :</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="cb_screen">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>0, width 40</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>0, width 80</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>not tile based</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Name Table:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="le_nametable">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Pattern Generator Table :</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="le_patterntable">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Color Table:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="le_colortable">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Number of lines:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QComboBox" name="cb_screenrows">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>In TEXT2 and GRAPH3 extra lines can be displayed</string>
               </property>
               <item>
                <property name="text">
                 <string>24</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>26.5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>32</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QCheckBox" name="cb_blinkcolors">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>In TEXT2 alternate (blink)colors can be used</string>
               </property>
               <property name="text">
                <string>Use blink colors</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QCheckBox" name="cb_color0">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Normaly color 0 will be the border color, check this to use the actual color 0 from the VDP</string>
               </property>
               <property name="text">
                <string>color 0 not border</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QSpinBox" name="sp_bordercolor">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximum">
                <number>15</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Border color:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QCheckBox" name="useVDPPalette">
               <property name="text">
                <string>Use VDP  palette registers</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="Line" name="line">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>880</width>
           <height>0</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Pattern</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_clickedcharimage">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="plainText">
          <string>Click on the tilemap to show character info</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
